src/Admin/Controllers/LoginController.cs

```diff
@@ -75,8 +74,21 @@ public async Task<IActionResult> Logout()
             await _signInManager.SignOutAsync();
             return RedirectToAction("Index", new
             {
-                success = "You have been logged out."
+                success = 1
             });
         }
+
+        private string GetMessage(int? messageCode)
```

Comment:
Consider internationalization of strings/messages; probably not a priority for this change since there's not a change in constructs, just consolidation of messages.

Reply:
Do these message bubble all the way back up to the application layer? If so, I second this (for eventual implementation)

Reply:
Yes, we do not have any localization on the server-side at this time.